docs/docs/getting-started/installation/aws.md

```diff
@@ -128,15 +128,68 @@ Error creating vpc:  operation error EC2: CreateVpc, https response error Status
 When encountering this, you can delete some of the resources just as described
 on [here](/getting-started/installation/aws#uninstall-airy-core)
 
+## Enable HTTPS
+
+This section explains all the steps needed to configure HTTPS on your `Airy Core` instance.
+
+Create a self-signing certificate (skip step if you already have your existing https certificate).
+The certificates will be created in your `~/.airy/certs` directory, using the OpenSSL utility.
```

Comment:
is it necessary that the certs are in `~/.airy`? I can't say from reading the doc but if it is then we have to be more explicit, if not we should make it clear it's an example